var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import tempfile
+
+import llnl.util.tty as tty
+
+
+class PyJaxlib(PythonPackage, CudaPackage):
+    """XLA library for Jax"""
+
+    homepage = "https://github.com/google/jax"
+    url = "https://github.com/google/jax/archive/refs/tags/jaxlib-v0.1.74.tar.gz"
+
+    tmp_path = ""
+
+    version('0.1.74', sha256='bbc78c7a4927012dcb1b7cd135c7521f782d7dad516a2401b56d3190f81afe35')
+
+    depends_on('python@3.7:')
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
+    depends_on('py-numpy@1.18:', type=('build', 'run'))
+    depends_on('py-scipy', type=('build', 'run'))
+    depends_on('py-absl-py', type=('build', 'run'))
+    depends_on('py-flatbuffers@1.12:2.999', type=('build', 'run'))
```

Comment:
Where do you see this dep listed?

Also, there are missing deps on opt-einsum and typing-extensions

Reply:
I think this is still setup.py for jax vs. jaxlib confusion?

Reply:
I don't think so, I just downloaded the tarball listed above and it isn't in the `setup.py`